sdk/ml/azure-ai-ml/.pre-commit-config.yaml

```diff
@@ -26,8 +26,8 @@ repos:
     args: ['--settings-file', 'sdk/ml/azure-ai-ml/pyproject.toml']
 - repo: local
   hooks:
-    - id: pylint
-      name: pylint
+    - id: pylint-dependencies-check
+      name: pylint-dependencies-check
```

Comment:
What's the rationale behind splitting this into two steps?

Reply:
the `pylint` in combined check has some error in applying a correct rcfile in some cases

Reply:
I'm not exactly sure how splitting this into two steps would resolve finding the correct rcfile, but I'm guessing I'm missing something.

---

I'm not interested in blocking your PR over this, so feel free to merge if you need to.

Reply:
Thanks, at least it won't fail for now. Will involve you next time I'm blocked by this